src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CATEGORIES_STATUS = 'CATEGORIES_STATUS';
```

Comment:
As suggested above and for the same reasons, kindly utilize the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for defining your action types.

Reply:
Noted with thanks